components/calendar/events.md

```diff
@@ -76,6 +78,72 @@ When handling the `ViewChanged` event, you cannot use two-way binding for the `V
 
 >tip You are not required to provide an initial value to the `View` parameter. It will default to `CalendarView.Month`.
 
+## RangeStartChanged
+
+The `RangeStartChanged` fires every time the user selects a new starting date for the range of dates. The first click on a date in the Calendar will always be the starting date and the range can be consisted of dates only onwards.
+
+## RangeEndChanged
+
+The `RangeEndChanged` fires every time when the selection of dates is finished.
+
+>caption Example of `Range` Selection with `RangeStartChanged` and `RangeEndChanged` events
+
+````CSHTML
+@* Observe the behavior of the RangeStartChanged and RangeEndChanged events and adding the selected dates to a List *@
+
+<div class="example-wrapper">
+    <h4>Range Selection Initial Selection</h4>
+
+    <TelerikCalendar Views="3"
+                     Date="@Date"
+                     RangeStart="@RangeStart"
+                     RangeEnd="@RangeEnd"
+                     SelectionMode="@CalendarSelectionMode.Range"
+                     RangeStartChanged="@StartChangeHandler"
+                     RangeEndChanged="@EndChangeHandler">
+    </TelerikCalendar>
+</div>
+
+@if (SelectedDates.Any())
+{
+    <div class="mt-3">
+        <h5 class="text-info">Selected dates:</h5>
+        @foreach (var day in SelectedDates)
+        {
+            <p class="text-muted">@day</p>
+        }
+    </div>
+}
+
+@code {
+    public DateTime Date { get; set; } = new DateTime(2020, 3, 1);
+    public DateTime RangeStart { get; set; } = new DateTime(2020, 3, 14);
+    public DateTime RangeEnd { get; set; } = new DateTime(2020, 3, 18);
+    public List<DateTime> SelectedDates { get; set; } = new List<DateTime>();
+
+    public void StartChangeHandler(DateTime startDate)
+    {
```

Comment:
Add comments about changing the view model or else the user action will be cancelled. See other similar events for inspiration

Reply:
Done with: 6517076ff1c6a5890590a43c1e44753654ca32e0